base_geoengine/geo_model.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2011-2012 Nicolas Bessi (Camptocamp SA)
+# Copyright 2016 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, models, tools
+from odoo.exceptions import except_orm, MissingError
+from . import geo_operators
+from . import fields as geo_fields
+
+
+DEFAULT_EXTENT = ('-123164.85222423, 5574694.9538936, '
+                  '1578017.6490538, 6186191.1800898')
+
+
+class GeoModel(models.BaseModel):
```

Comment:
Do we still need this specialized model? Why not inheriting from `base`?

Reply:
I think the same it was not a priority but I agree we can improve this. I'll add it in the todo list.